pipeline/outputs/bigquery.md

```diff
@@ -60,6 +60,7 @@ You must configure workload identity federation in GCP before using it with Flue
 | provider\_id | GCP workload identity provider. Used to construct the full resource name of the identity provider. Currently only AWS accounts are supported. |  |
 | google\_service\_account | Email address of the Google service account to impersonate. The workload identity provider must have permissions to impersonate this service account, and the service account must have permissions to access Google BigQuery resources (e.g. `write` access to tables) |  |
 | workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `0` |
+| buffer_size | Sets the HTTP client buffer size (in bytes) for reading API responses from BigQuery. This is a fixed-size allocation; responses larger than this size will trigger a warning. | `4192` |
```

Comment:
Well, I presume the warning is that they won't be handled right or is this saying it will auto-grow?